Resources/en.lproj/Localizable.strings

```diff
@@ -1153,13 +1154,22 @@
 "mapil_pano_only" = "Display only 360° images";
 "mapil_select_user" = "Select user";
 
+// Coordinate widget
+"coordinates_widget" = "Coordinates widget";
+"searching_gps" = "Searching GPS";
+"copied_to_clipboard" = "Copied to clipboard";
+
 // Quick action
 "quick_action_name" = "Quick action";
 "quick_action_no_overlay" = "No overlay";
 "quick_action_add_overlay" = "Add overlay";
 "quick_action_no_underlay" = "No underlay";
 "quick_action_add_underlay" = "Add underlay";
 "quick_action_list_descr" = "Tapping this action button pages through the list below";
+"toggle_coordinates" = "Show/hide coordinates widget";
+"show_coordinates" = "Show coordinates";
+"hide_coordinates" = "Hide coordinates";
+"quick_action_coordinates_widget_descr" = "A toggle to show or hide the Coordinates widget on the map.";
```

Comment:
Description should be written in the same style as existing actions.
Example:
"quick_action_hillshade_descr" = "Tapping this action button shows or hides the terrain layer.";